docs/index.rst

```diff
@@ -0,0 +1,40 @@
+.. nuclio documentation master file, created by
+   sphinx-quickstart on Wed Sep 20 10:10:53 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Nuclio documentation!
+==================================
+
```

Comment:
Add `Introduction`, see: https://nuclio.io/docs/latest/introduction/
It's basically the root README.md, can it be linked here or will it need to be duplicated?